docs/integrations/sigrid-api-documentation.md

```diff
@@ -468,6 +468,244 @@ This end point will return the following response structure:
+		"systemName" = "system-c"
+	}
+  ],
+  "lastLoginAt": "2024-03-07T16:54:33.438Z"
+}
+```
+
+Related to this is the `accessToAll` boolean field, which indicates if a user has the explicit right to access every system in the portfolio, both for systems that are currently onboarded and those that will be onboarded in the future. This permission is typically reserved for administrators of the Sigrid account, however this option can be applied to all users within a portfolio. 
+
+For more information, see the detailed section on [user management in Sigrid.](../organization-integration/usermanagement.md)
+
+### Managing authorization group permissions
+
+In addition to managing individual user permissions within the portfolio, authorization group permissions can also be managed via the API. This allows for bulk editing of permissions users inherit based on their authorization group membership, as well as the set of users that are part of a specified group. 
+
+Available endpoints include:
+- `GET https://sigrid-says.com/rest/auth/api/user-management/{customer}/groups`: Returns a ist all authorization groups defined within a portfolio
+- `GET https://sigrid-says.com/rest/auth/api/user-management/{customer}/groups/{groupId}`: Returns an authorization group based on their unique identifier
+- `PUT https://sigrid-says.com/rest/auth/api/user-management/{customer}/groups/{groupID}/permissions`: Updates the permissions granted to the authorization group, with all users within the group inheriting the updated permissions
+- `PUT https://sigrid-says.com/rest/auth/api/user-management/{customer}/groups/{groupID}/members`: Updates the set of users that are part of the authorization group
+
+The path parameters `{customer}` and `{groupID}` refer to your Sigrid account name and a unique authorization group ID respectively.
+
+The response format upon successful request of a single authorization group looks like the following:
+
+```json
+{
+  "id": "string($uuid)",
+  "name": "string",
+  "description": "string",
+  "users": [
+    "string($uuid)"
+  ],
+  "systems": [
+    {
+      "systemName": "string"
+    }
+  ],
+  "updatedAt": "2024-03-07T17:41:59.278Z",
+  "updatedByUser": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
+}
+```
+
+Unlike the response received when leveraging the endpoint to return a single user, groups have two fields that can be modified via the API: `users` and `systems`. 
+
+- The `users` field contains the set of users that are members of the authorization group and will inherit the permissions granted to the group. A user is denoted by a UUID representing their unique identifier hash.
+- The `systems`field contains the set of systems authorized to be accessible by the group, similar to how it is defined for an individual user. System are denoted by their system name when onboarded in sigrid. 
+    - __NOTE:__ This name can be different from the display name seen in Sigrid, be sure to use the name the sytem was attributed during onboarding. 
```

Comment:
I would remove the double indentation for this NOTE

Reply:
It directly relates to the previous line, so I'm more in favor of nesting it here. Makes it more explicit what name we're referring to.

Reply:
Ok 👍